src/main/java/dk/kb/datahandler/oai/OaiResponseFilterPreservica.java

```diff
@@ -0,0 +1,84 @@
+/*
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package dk.kb.datahandler.oai;
+
+import dk.kb.storage.model.v1.DsRecordDto;
+import dk.kb.storage.model.v1.RecordTypeDto;
+import dk.kb.storage.util.DsStorageClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Filtering and delivery of Preservica OAI records. Generates {@code datasource} prefixed IDs,
```

Comment:
These records should be origin prefixed right? As we are splitting the datasource into two origins in this implementation

Reply:
Oh, nevermind. This feature haven't been implemented yet here!